j/js-cookie/js-cookie_ubi_8.5.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package	: js-cookie
+# Version	: v3.0.0
+# Source repo	: https://github.com/js-cookie/js-cookie
+# Tested on	: UBI: 8.5
+# Language      : Node
+# Travis-Check  : True
+# Script License: MIT License
+# Maintainer	: Vishaka Desai <Vedang.Wartikar@ibm.com>
+#
+# Disclaimer: This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=js-cookie
+PACKAGE_VERSION=${1:-v3.0.0}
+PACKAGE_URL=https://github.com/js-cookie/js-cookie.git
+
+yum -y update && yum install -y yum-utils nodejs npm git make wget tar zip nss libXScrnSaver libX11-xcb libXcomposite libXcursor libXfixes libXrender libXdamage atk at-spi2-atk at-spi2-core cups-libs avahi-libs libXrandr libdrm mesa-libgbm libwayland-server alsa-lib pango gtk3
```

Comment:
yum update should not be needed, also for readability split into two lines so that it does not exceed 80 characters on one line